libraries/Preferences/src/Preferences.cpp

```diff
@@ -53,10 +53,14 @@ bool Preferences::begin(const char * name, bool readOnly, const char* partition_
     return true;
 }
 
-void Preferences::end(){
+void Preferences::end(){                                         // modified to add an nvs_commit()
```

Comment:
The comment isn't necessary, as the file change history is visible in `git log` or `git blame`, and any commit that has modified a file can be found and reverted if needed. The comments describing the changes (`// added by XXX 06/2021`) only distract from the purpose of the code.